src/lib.rs

```diff
@@ -177,7 +197,21 @@ cfg_if! {
         }
     } else {
         unsafe fn guess_os_stack_limit() -> usize {
-            panic!("cannot guess the stack limit on this platform");
+            0
+        }
+        mod exports {
+            #[no_mangle]
+            extern fn __stacker_stack_pointer() -> usize {
```

Comment:
I think these may be best compiled as fallback shims in `build.rs`, because these aren't always available by target os but also by target arch (aka powerpc implementations don't exist)